examples_tests/61.BlitFilterTest/normalization/normalization.glsl

```diff
@@ -0,0 +1,92 @@
+#ifndef _NBL_GLSL_BLIT_NORMALIZATION_INCLUDED_
+#define _NBL_GLSL_BLIT_NORMALIZATION_INCLUDED_
+
+#include <nbl/builtin/glsl/workgroup/arithmetic.glsl>
+
+uint integerDivide_64_32_32(in uint dividendMsb, in uint dividendLsb, in uint divisor)
+{
+	const uint MAX_UINT = ~0u;
+	const float msbRatio = float(dividendMsb) / float(divisor);
```

Comment:
this line looks really off

Reply:
Pretty thoroughly tested for dividend (combined result of `dividendMsb` and `dividendLsb`) <= `2^56`.
After that, due to floating point precision, `msbRatio` sometimes starts to come out incorrect.
I think you can also copy and paste it into a cpp file and verify my claims :)

I was able to use some simplifying assumptions here:
- the fact that it is an integer divide
- the fact that output (quotient) will always be 32 bits
to churn this out pretty quickly.

If dividend > `2^56` would be regular occurrence for us, we can spend some more time here to make this robust.

Reply:
ok, just put a big fat comment that it will break when dividendMsb>23

Reply:
as soon as that big fat comment is there I'll mark resolved.

Reply:
did it myself